cva6/tests/custom/sv32/vm_mstatus_tvm_01.S

```diff
@@ -0,0 +1,176 @@
+#=======================================================================
+#      RW Access of satp and sfence.vma in User and Supervisor mode when 
+#      mstatus.tvm = 1  
+#-----------------------------------------------------------------------
+# Test Description:
+# If mstatus.TVM=1, read and write access to the satp and sfence.vma will 
+# raise illegal instruction exception in S-mode. This test
+# covers the following scenarios in both supervisor and user privilege
+# modes for level1 PTE.
+#                           Level 1
+#   - mstatus.tvm = 1 and test the RW access of satp and sfence in 
+#     in supervisor and user mode
```

Comment:
Testing only s mode will be enough for this test

Reply:
ok I will update the test accordingly.